src/components/PublisherInfo.tsx

```diff
@@ -0,0 +1,48 @@
+import Link from 'next/link';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Avatar from 'react-avatar';
+import Image from 'next/image';
+
+function PublisherInfo({
+  userData,
+  section,
+}: {
+  userData: {
+    id: number;
+    name: string;
+    username: string;
+    profilePicture: string;
+    membership: string;
+  };
+  section: { SectionName: string; SectionUrl: string; SectionPhoto: string };
+}) {
+  return (
+    <div>
+      <div className="flex">
+        <button type="button">
+          <Avatar title="test" src={userData.profilePicture} size="60" round />
```

Comment:
same as the comment about the `DOMPurify`, you should not use react avatar directly in here

instead create a new component that we can call Avatar, for this component can only return react avatar directly

in the future we will want to add some customaizations to the avatar so we could do that in one place